fs/walk.ts

```diff
@@ -97,27 +105,31 @@ export async function* walk(
     const filename = join(root, info.name!);
 
     if (info.isFile()) {
-      if (include(filename, options)) {
+      if (options.includeFiles != false && include(filename, options)) {
         yield { filename, info };
       }
     } else {
-      if (!(options.maxDepth! < 0)) {
-        yield* walk(filename, options);
-      }
+      yield* walk(filename, { ...options, maxDepth: maxDepth - 1 });
     }
   }
 }
 
 /** Same as walk() but uses synchronous ops */
 export function* walkSync(
-  root: string = ".",
```

Comment:
why?

Reply:
`walk()` didn't have this so something was wrong, I chose to change this one for no particular reason. I guess they could both be changed to have "." as a default but IMO root would have to go in the options object then so this was just the smaller change.